webrtc/RTCDataChannel-bufferedAmountLowThreshold.html

```diff
@@ -0,0 +1,181 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCDataChannel.bufferedAmountLowThreshold</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="RTCPeerConnection-helper.js"></script>
+<script>
+'use strict';
+
+// Disable global timeout
+// IMPORTANT: You need to add a timeout to *every test* of this file!
+setup({ explicit_timeout: true });
```

Comment:
Why is the default timeout inappropriate here?
I see that a 5 second timeout is used, while the default timeout is 10 seconds. Saving 5 seconds in the failure case seems hardly worth doing.

Reply:
The problem is that there's a varying amount of test cases onto which one constant global timeout is applied, so it's 10 or 60 seconds globally for all test cases no matter how many you have. This workaround allows us to apply a local timeout to each test case.